### PR TITLE
feat: TT-332 Dark mode 

### PR DESCRIPTION
**Pull request by JosueOb**
# Description

The dark mode is a temporary change in the color palette of the software, this feature offers several benefits, among the main ones are: it prevents the user's vision from getting tired in less time and battery-saving due to the reduction of the screen brightness. 

# Solution
Using Tailwind CSS to style Time Tracker UI in dark mode, allowing the user to switch between light and dark mode. Also, Tailwind was configured to set the color palette for the project.

### Light mode
image

### Dark mode
image

### CODE REVIEW

**Comment by DrFreud1** [comment]
Hello Jose, I am not sure if you have noticed, but the content from certain entries is not visible in most tabs, with exception of the Time Clock tab. Also, I see that the test coverage has not reached the expected minimum with the current changes. 
On the other hand, I think that the color palette is very well implemented and looks nice.

**Review by EdansRocks — APPROVED** [review]

**Review by DrFreud1 — APPROVED** [review]
Good Job @JosueOb !

File: src/app/modules/shared/components/sidebar/sidebar.component.scss [hunk 1]
```diff
@@ -62,3 +63,9 @@ body {
   height: -o-calc(100vh - 12vh);
   height: calc(100vh - 12vh);
 }
+
+.dark-mode-toggle{
```
Comment (scastillo-jp) [inline src/app/modules/shared/components/sidebar/sidebar.component.scss:67]: add a space after **.dark-mode-toggle**

`.dark-mode-toggle {`
